### PR TITLE
Black pepper tofu

### PR DESCRIPTION
**Pull request by matthewdking**
relates #40

### CODE REVIEW

File: recipes/main/black-pepper-tofu.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Black pepper tofu
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-60%25-orange.svg)
```
Comment (finnhodgkin) [inline recipes/main/black-pepper-tofu.md:2]: This badge is broken

**Update** After a refresh it appears to be working. It's not a repeatable bug but the url looks a little weird so there might be something wrong :woman_shrugging:

File: recipes/main/black-pepper-tofu.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**First cooked by:** [@matthewdking](https://github.com/matthewdking/)
+
+**Date:** Week 16 FACN2
+
+**Author of Recipe**: [@matthewdking](https://github.com/matthewdking/)
+
+This tasty tofu recipe is a hit with everyone and goes well with rice or noodles. Side dishes are a good idea such as some roasted red peppers or mushrooms.
```
Comment (finnhodgkin) [inline recipes/main/black-pepper-tofu.md]: You showed me a delicious looking photos of the recipe that would look great in the readme :+1:

File: recipes/main/black-pepper-tofu.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Average-orange.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**First cooked by:** [@matthewdking](https://github.com/matthewdking/)
+
+**Date:** Week 16 FACN2
+
+**Author of Recipe**: [@matthewdking](https://github.com/matthewdking/)
```
Comment (finnhodgkin) [inline recipes/main/black-pepper-tofu.md]: A nod to the original author ([Yotam Ottolenghi](https://www.ottolenghi.co.uk/black-pepper-tofu-a-shop)) is always appreciated. I for one think he'd be flattered to be included in such a prestigious cooking forum.

File: recipes/main/black-pepper-tofu.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+This tasty tofu recipe is a hit with everyone and goes well with rice or noodles. Side dishes are a good idea such as some roasted red peppers or mushrooms.
+
+## Ingredients
+
+### For the beans
++ tofu (four packs)
```
Comment (finnhodgkin) [inline recipes/main/black-pepper-tofu.md]: Packs in the UK vary in size quite a lot. Definitely not a block to merging though :+1:

**Review by finnhodgkin — CHANGES_REQUESTED** [review]

File: recipes/main/black-pepper-tofu.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
++ cornflour
++ corn oil
++ 3 chillies
++ 3cm ginger
++ 1 large white onion
++ 1 bulb of garlic
```
Comment (m4v15) [inline recipes/main/black-pepper-tofu.md]: Was it actually one bulb of garlic?! Like I believe you I'm just wanted to express my surprise

File: recipes/main/black-pepper-tofu.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
++ tablespoon caster sugar
++ small bunch of spring onions
+
+## Method
+
+### For the tofu
++ Cut the tofu into small 1-2cm cubes
++ Coat each tofu cube with corn flour
++ Once coated shallow fry in corn oil the tofu until crispy
++ Be careful not to fry too many in a pan at once as they can start to stew
```
Comment (m4v15) [inline recipes/main/black-pepper-tofu.md:41]: nice tip 👍

File: recipes/main/black-pepper-tofu.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+### For the beans
++ tofu (four packs)
++ cornflour
++ corn oil
++ 3 chillies
++ 3cm ginger
++ 1 large white onion
++ 1 bulb of garlic
++ dark soy sauce
++ sweet soy sauce
```
Comment (m4v15) [inline recipes/main/black-pepper-tofu.md]: I would say normally with soy sauce you don't need a measurement but with this one I remember you used a lot so maybe some guidance here would be helpful

File: recipes/main/black-pepper-tofu.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+## Method
+
+### For the tofu
++ Cut the tofu into small 1-2cm cubes
++ Coat each tofu cube with corn flour
++ Once coated shallow fry in corn oil the tofu until crispy
++ Be careful not to fry too many in a pan at once as they can start to stew
++ Set aside on some kitchen roll to soak up some oil
+
+### Rice
```
Comment (m4v15) [inline recipes/main/black-pepper-tofu.md]: Could change this to Rice/Noodles/Side Dish?

Although don't know if you need this line at all, the line in the intro does enough? Most recipes don't have a part saying make sure the other stuff is ready at the same time

File: recipes/main/black-pepper-tofu.md (hunk 3 above)
Comment (mattlub) [reply recipes/main/black-pepper-tofu.md]: Does he have GitHub though???

**Review by m4v15 — CHANGES_REQUESTED** [review]
maybe don't need to change, just had see my questions

File: recipes/main/black-pepper-tofu.md (hunk 5 above)
Comment (matthewdking) [reply recipes/main/black-pepper-tofu.md]: I think it was more like half! Maybe I will add a specific number of cloves

**Comment by bradreeder** [comment]
I'm making this now (thanks, and hi from London 😋 ) and I don't think there's anything in the recipe telling you what to do with the chillies? (Assuming to add to the sauce along with the onion, garlic, and ginger.)

**Comment by matthewdking** [comment]
@bradreeder good spot! Remove the seeds from the chillies and then finely chop them and add them with the onion garlic and ginger. I will add it to the recipe now! Hope you enjoy it :man_cook: :fork_and_knife:

**Comment by bradreeder** [comment]
@matthewdking I think I went overboard on the chillies 🔥 🚒 but v nice thanks, have enough leftover for the weekend.

**Comment by yvonne-liu** [comment]
@bradreeder you should star this repo ⭐️ !

File: recipes/main/black-pepper-tofu.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**First cooked by:** [@matthewdking](https://github.com/matthewdking/)
+
+**Date:** Week 16 FACN2
+
+**Author of Recipe**: [Yotam Ottolenghi](https://www.ottolenghi.co.uk/black-pepper-tofu-a-shop)
+
+This tasty tofu recipe is a hit with everyone and goes well with rice or noodles. Side dishes are a good idea such as some roasted red peppers or mushrooms.
```
Comment (Karyum) [inline recipes/main/black-pepper-tofu.md]: I'm not sure you could say it was a hit with everyone, thats just a broad statement.

File: recipes/main/black-pepper-tofu.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+
+
+## Ingredients
+
+### For the beans
++ firm tofu (four packs)
++ cornflour
```
Comment (Karyum) [inline recipes/main/black-pepper-tofu.md]: on line 39 you have corn flour with a pace between them.
consistency

**Review by Karyum — CHANGES_REQUESTED** [review]
Also add the links to the README

File: recipes/main/black-pepper-tofu.md (hunk 10 above)
Comment (yvonne-liu) [reply recipes/main/black-pepper-tofu.md]: What is a pace?

Comment (m4v15) [reply recipes/main/black-pepper-tofu.md]: he's right tbf, should be cornflour everywhere

Comment (Karyum) [reply recipes/main/black-pepper-tofu.md]: @yvonne-liu I meant space but 
http://www.dictionary.com/browse/pace

File: recipes/main/black-pepper-tofu.md (hunk 9 above)
Comment (matthewdking) [reply recipes/main/black-pepper-tofu.md]: Who was it not a hit with?

Comment (Karyum) [reply recipes/main/black-pepper-tofu.md]: well me atleast

**Review by Karyum — APPROVED** [review]

**Review by finnhodgkin — APPROVED** [review]

File: README.md [hunk 11]
```diff
@@ -7,6 +7,8 @@ You know what this is for.
 
 [Black beans w/ Lime & Coriander rice (and sides)](./recipes/main/black-beans.md)
 
+[Black pepper tofu](./recipes/main/black-pepper-tofu.md)
```
Comment (yvonne-liu) [inline README.md]: The inconsistency with capitalizing with the rest of the names is 🙅 .

**Review by yvonne-liu — CHANGES_REQUESTED** [review]

File: README.md (hunk 11 above)
Comment (yvonne-liu) [reply README.md]: Might as well change it to "Black Beans with Lime and Coriander Rice" as well.

**Review by yvonne-liu — APPROVED** [review]

**Comment by m4v15** [comment]
@yvonne-liu errr you should wait until everyone has approved the pull request before merging m8

**Comment by yvonne-liu** [comment]
@m4v15 "Once a pull request has been reviewed by 3 people who ate the meal, as well as or including a maintainer, it can be merged." 👋

**Comment by m4v15** [comment]
@yvonne-liu Firstly, that comes under the section titles "Instructions for maintainers" which you are not. Second of all, even if it didn't, the "3 people who ate the meal" criteria wasn't fulfilled here as @finnhodgkin did not, and *third* of all, I would *think* it is safe to assume/implied knowledge that you don't merge a PR when someone has requested changes and is yet to approve it.

**Comment by m4v15** [comment]
I mean, I was joking at first, but now this is serious

**Comment by m4v15** [comment]
Oh also I mean just cos I had requested changes and not reviewed it, not all the other people who hadn't reviewed it

**Comment by m4v15** [comment]
you can apologise here or on WhatsApp, either works ❤️

**Comment by yvonne-liu** [comment]
I'd _consider_ apologizing, but I'll never apologise 💔

**Comment by m4v15** [comment]
that'll do

**Comment by mattlub** [comment]
> you don't merge a PR when someone has requested changes and is yet to approve it

@yvonne-liu agree with @m4v15 about this, should it be stated in the guidelines?

**Pull request merged**